### DISCUSSION TITLE
Really looking forward to Item info for POE2.

### DISCUSSION THREAD

**Discussion by sclient** [Lailloken/Exile-UI#490] [Ideas & Feedback]
As you know, the in game advanced tooltip stuff is miserably transparent and cannot be adjusted, and ofc has never had the kind of DPS breakdowns Lai offered, so it is probably the biggest feature on my wanted list. I wonder if the roadmap guy will have it for next league start (and ofc what POB 2 will look like).

**Comment by Lailloken** [comment]
The item-info tooltip was one of the first things I wanted to port to PoE2, but I noticed that they removed the ability to copy advanced item information in game. Right now, holding ALT only shows more information in game, but that extra information is not being copied into the clipboard.
I started porting over the feature yesterday anyway to at least have icons and the ability to mark mods as (un)desired, so here's what it'll look like for the foreseeable future (I might release an update today):
![image](https://github.com/user-attachments/assets/67901074-2184-40f5-9a97-067e9ee7b7fc)

As you can see, clipboard information doesn't include mod tiers, rolls, affix types, mod groups, etc. Right now, you can't even tell if there are hybrid mods on an item.

**Comment by sclient** [comment]
Well, its a start! I guess we will have to wait on GGG for this one, but thanks for the effort! :)
